blocks/library/embed/index.js

```diff
@@ -515,13 +596,15 @@ export const others = [
 			title: 'VideoPress',
 			icon: 'embed-video',
 			keywords: [ __( 'video' ) ],
+			patterns: [ 'videopress\.com\/.+' ],
 		} ),
 	},
 	{
 		name: 'core-embed/wordpress-tv',
 		settings: getEmbedBlockSettings( {
 			title: 'WordPress.tv',
 			icon: 'embed-video',
+			patterns: [ 'wordpress\.tv\/.+' ],
```

Comment:
Yes, I put all these in by hand. Yes, I'm comfortable with that. I am enlightened. Thanks for reading this far!